tests/validators/test_dict.py

```diff
@@ -216,3 +217,58 @@ def test_dict_length_constraints(kwargs: Dict[str, Any], input_value, expected):
             v.validate_python(input_value)
     else:
         assert v.validate_python(input_value) == expected
+
+
+@pytest.mark.skipif(platform.python_implementation() == 'PyPy', reason='dict views not implemented in pyo3 for pypy')
```

Comment:
Sorry, but it'll be confusing if a test for list, tuple, set, frozen set is in the file for dict tests.

Please can you copy this to each of those files and simplify the parameterisation accordingly.

Reply:
No worries. I thought it would be easier to consolidate them all in a single test, but totally happy to make this change.

Reply:
I understand, but still think better in this case.